pkg/imrpc/proxy.proto

```diff
@@ -214,6 +216,13 @@ message EngineReplicaRemoveRequest {
 	string replica_address = 2;
 }
 
+message EngineReplicaUpdateRequest {
```

Comment:
1. This is more like `ReplicaModeUpdate`
2. The field `mode` is a controller in-memory field. Should replicas understand it?

Reply:
Updated. Thank you.